packages/payment-processor/src/payment/erc777-stream.ts

```diff
@@ -0,0 +1,55 @@
+import { ContractTransaction, Signer } from 'ethers';
+
+import { ClientTypes, ExtensionTypes, PaymentTypes } from '@requestnetwork/types';
+
+import {
+  getPaymentNetworkExtension,
+  getProvider,
+  getRequestPaymentValues,
+  validateRequest,
+} from './utils';
+import { Framework } from '@superfluid-finance/sdk-core';
+
+/**
+ * Processes a transaction to pay an ERC777 stream Request.
+ * @param request
+ * @param signerOrProvider the Web3 provider, or signer. Defaults to window.ethereum.
+ * @param _overrides optionally, override default transaction values, like gas.
+ */
+export async function payErc777StreamRequest(
+  request: ClientTypes.IRequestData,
+  signer: Signer,
+): Promise<ContractTransaction> {
```

Comment:
The JSDoc does not correspond to the method signature

Reply:
I'm adding again the optional overrides to make it coherent with other payment processor, I wanted to have a simpler version first.